src/time.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of FreeRCT.
+ * FreeRCT is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * FreeRCT is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with FreeRCT. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file time.h Functions for measuring time. */
```

Comment:
Ah right. you may want to rename it to a filename that is not used in the standard library,

Also, I always tried to push as many `.h` files down into the `.cpp` files to avoid including too much for each file. Not sure if that is feasible here as well.